app/assets/javascripts/discourse/app/components/composer-body.js

```diff
@@ -62,6 +62,16 @@ export default Component.extend(KeyEnterEscape, {
     this.appEvents.trigger("composer:resized");
   },
 
+  @observes("composeState")
+  _setMainOutletPaddingBottom() {
+    const mainOutletElement = document.querySelector("#main-outlet");
+    if (this.composeState === Composer.OPEN) {
+      mainOutletElement.style.paddingBottom = "300px";
+    } else if (this.composeState === Composer.CLOSED) {
+      mainOutletElement.style.paddingBottom = "0";
```

Comment:
```suggestion
      mainOutletElement.style.paddingBottom = "";
```

Better to empty the inline value, so that if a stylesheet has a value, it would still apply.